docs/iris/example_code/Meteorology/wind_speed.py

```diff
@@ -0,0 +1,77 @@
+"""
+Plotting wind direction using quiver
+===========================================================
+
+This example demonstrates using quiver to plot wind speed
+
+The second plot uses normalised data to produce arrows with a uniform size
```

Comment:
For the second plot, the data used for the arrows is normalised to produce arrows with a uniform size on the plot.